packages/js/storybook/bin/storybook.js

```diff
@@ -3,11 +3,10 @@
  * External dependencies
  */
 import { argv } from 'process';
+import shell from 'shelljs';
+import path from 'path';
 
-/**
- * Internal dependencies
- */
-import { shell } from '../utils.js'
+process.env.PATH += ( path.delimiter + path.join( process.cwd(), 'node_modules', '.bin') );
```

Comment:
This line is repeated over all the bin files, so I guess could be reuse it in `utils.js` kinda file.

Reply:
In all 2 ;) We cannot.
as npm fails to resolve shelljs dependency in a parent folder without packageits own .json
https://github.com/woocommerce/grow/pull/10/commits/99141963a638199522a213fdda065f05b7847a9a